.editorconfig

```diff
@@ -63,6 +63,10 @@ end_of_line = lf
 [Makefile]
 indent_style = tab
 
+# XML Resource Files

```

Comment:
Could we move this below `# XML Configuration Files`, so that the XML file config is grouped together?